src/sentry/utils/performance_issues/detectors/consecutive_http_detector.py

```diff
@@ -70,6 +71,14 @@ def _validate_and_store_performance_problem(self):
     def _store_performance_problem(self) -> None:
         fingerprint = self._fingerprint()
         offender_span_ids = [span.get("span_id", None) for span in self.consecutive_http_spans]
+        offender_span_evidence_display = [
+            IssueEvidence(
+                name="Offending Spans",
```

Comment:
is that what the label should read?

Reply:
Hmm good point. Its not necessarily what it should read. I'm going to update this to "Consecutive Span" as it's more accurate. 
This detector is only being prepared for EA, this might change going from EA -> GA when we finalize design/naming details.

Reply:
we won't be using `evidence_display` for perf issues. feel free to return an empty list instead